covigator/database/model.py

```diff
@@ -1150,3 +1152,43 @@ class Lineages(Base):
     variant_under_investigation = Column(Boolean, default=False)
     parent_lineage_id = Column(String)
     tags = Column(String)
+
+    variants = relationship(
+        "LineageDefiningVariants",
+        secondary=LINEAGE_SITES_JUNCTION_TABLE_NAME,
+        back_populates='constellations',
+    )
+
+
+class LineageDefiningVariants(Base):
+    """
+    Store lineage defining mutations
+    """
+    __tablename__ = CONSTELLATION_SITES_TABLE_NAME
+
+    variant_id = Column(String, primary_key=True)
+    variant_type = Column(String)
```

Comment:
You could use `variant_type = Column(Enum(VariantType, name=VariantType.__constraint_name__))` as in other variant tables